scripts/go-to-slide.js

```diff
@@ -7,8 +7,10 @@ H5P.CoursePresentation.GoToSlide = (function ($) {
    * @class
    * @param {Number} slideNum
    * @param {CoursePresentation} cp
+   * @param {String} goToSlideNextPrevious
+   * @param {Number} index
    */
-  function GoToSlide(title, slideNum, invisible, cp) {
+  function GoToSlide(title, slideNum, invisible, cp, goToSlideNextPrevious, index) {
```

Comment:
_I guess for the labels it is better to let them say "next slide", "previous slide"?_
*shrug* I don't have strong feelings about what the labels are. Your call.

Are you suggesting the **values** should be "next slide" & "previous slide" instead of "next" & "previous"? 
Or change the **labels** from "Next slide" & "Previous slide" to "next slide" & "previous slide"?
I'm a bit confused. Feel free to tweak the labels/values or let me know what needs to change. Whatever is easier for you.